### ISSUE TITLE
Spec file doesn't generate configure script before build

### ISSUE THREAD

**Issue by ananace** [netdata/netdata#4570]
##### Bug report summary

The rpmspec that's provided in master and in the 1.11.0 release doesn't generate a configure script as part of the build steps, therefore failing to run configure as part of the build.

##### OS / Environment

Verified on CentOS 6 and CentOS 7, both on physical machines as well as in Docker containers.

##### Component Name

netdata

##### Steps To Reproduce

Download the 1.11.0 release, unpack the tarball, generate the spec file, set up an rpmbuild tree, copy the tarball into the sources directory - the spec into the spec folder, try to build the RPM.

##### Expected behavior

The rpmspec results in a usable RPM package.

**Comment by ktsaou** [comment]
How do you generate the RPM file.

I thought the right way was to something like this: https://github.com/netdata/netdata/blob/master/contrib/rhel/build-netdata-rpm.sh

I use that script to test that an RPM file can be generated. Is there a more automated way?

**Comment by ananace** [comment]
In my case, due to how the internal RPM building system is set up I have to use a plain spec file instead of generating and passing along a source RPM.

The problem comes if you take the netdata.spec.in file, fill in the placeholders with the release-specific values - be it by automake/script or just with sed, and then try to use said spec to build an RPM.  
The URL specified as source doesn't include a pre-built configure script, so the spec file's build steps aren't possible to complete.

**Comment by ktsaou** [comment]
ok, thanks. Let's wait for @paulfantom 	! All these are not my expertise.

**Comment by paulfantom** [comment]
This cannot be fixed in `1.11.0` as we need to modify files inside already released package. #4625 fixed that on a master branch so we can do a bugfix release ex. `1.11.1` with a fix (@ktsaou wdyt?).

TL;DR Problem was in `configure.ac` and it is solved in current master branch.

**Comment by ktsaou** [comment]
@paulfantom 	is this now fixed?

**Comment by paulfantom** [comment]
No, it is not fixed. This needs one more change in `configure.ac` - remove `_rolling` suffix from version string.
